LunchTime/Managers/LunchProvider.cs

```diff
@@ -3,92 +3,49 @@
 using System.Diagnostics;
 using System.Linq;
 using System.Net;
+using log4net;
 using LunchTime.Interfaces;
 using LunchTime.Models;
-using LunchTime.Shared;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Mvc.Rendering;
 using Newtonsoft.Json;
 
 namespace LunchTime.Managers
 {
     public class LunchProvider : ILunchProvider
     {
-        private readonly IHttpContextAccessor _httpContextAccessor;
+        private static readonly ILog _log = LogManager.GetLogger(typeof(MenusProvider));
```

Comment:
Generaly, it would be better to have the logger injected (Mocking for unit testing)

Reply:
changed in next commit